pkg/apiserver/apiserver.go

```diff
@@ -166,6 +167,7 @@ func (c completedConfig) New() (*WardleServer, error) {
 	v1beta1storage["applicationprofilesummaries"] = sbomregistry.RESTInPeace(applicationprofilesummary.NewREST(Scheme, storageImpl, c.GenericConfig.RESTOptionsGetter))
 	v1beta1storage["applicationactivities"] = sbomregistry.RESTInPeace(applicationactivity.NewREST(Scheme, storageImpl, c.GenericConfig.RESTOptionsGetter))
 
+	v1beta1storage["networkneighborses"] = sbomregistry.RESTInPeace(networkneighbors.NewREST(Scheme, storageImpl, c.GenericConfig.RESTOptionsGetter))
```

Comment:
chore: if using only `networkneighbors` as a resource qualifier worked, we have to use `networkneighbors` here, too